pom.xml

```diff
@@ -44,11 +44,9 @@
 
     <distributionManagement>
         <repository>
-            <uniqueVersion>false</uniqueVersion>
             <id>github</id>
             <name>GitHub Packages</name>
-            <url>https://maven.pkg.github.com/MarginallyClever/NodeGraphCore</url>
-            <layout>default</layout>
+            <url>https://maven.pkg.github.com/coliss86/nodegraphcore</url>
```

Comment:
probably leave that where it was?

Reply:
it is for the example, this PR isn't meant to be merged because the workflow is also modified and I don't know what you want to do. I'll close it soon when you have picked up the modifications